gulpfile.js

```diff
@@ -16,13 +16,15 @@ var info = require('./package.json');
 var request = require('request');
 
 var isWin = process.platform.indexOf('win') > -1;
+var isMac = process.platform.indexOf('darwin') > -1;
+var isLinux = process.platform.indexOf('linux') > -1;
```

Comment:
Would you be open to switching these lines to the same logic as https://github.com/davejm/p5.js-editor/blob/a340cf90835b15b7097a545c3ec28b67f7e8fab3/public/index.html#L41-L43 ?